<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="rcm">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-1.72144234930616E-06 -0.000156964168878601 0.0531360217416827"
        rpy="0 0 0" />
      <mass
        value="0.548884585225384" />
      <inertia
        ixx="0.000511577282673926"
        ixy="1.33340305831271E-09"
        ixz="-1.45841744290592E-09"
        iyy="0.000644163901137436"
        iyz="-1.87300620641781E-09"
        izz="0.000639190130910955" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="Link1">
    <inertial>
      <origin
        xyz="-1.0426E-06 0.017456 0.033231"
        rpy="0 0 0" />
      <mass
        value="0.34082" />
      <inertia
        ixx="0.00024163"
        ixy="-1.1458E-12"
        ixz="-4.6996E-14"
        iyy="0.00025274"
        iyz="2.9218E-06"
        izz="0.00025121" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link1_joint"
    type="revolute">
    <origin
      xyz="0 0 0.098695"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="Link2_1">
    <inertial>
      <origin
        xyz="0.055013 -5.3701E-12 0.00034718"
        rpy="0 0 0" />
      <mass
        value="0.044237" />
      <inertia
        ixx="4.5719E-06"
        ixy="-1.4076E-10"
        ixz="2.5464E-10"
        iyy="4.1666E-05"
        iyz="-1.2118E-14"
        izz="3.8711E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link2_1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link2_1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link2_1_joint"
    type="revolute">
    <origin
      xyz="0 0 0.055554"
      rpy="-1.5708 0 0" />
    <parent
      link="Link1" />
    <child
      link="Link2_1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
    <mimic
      joint="Link2_1_joint"
      multiplier="1"
      offset="0" />
  </joint>
  <link
    name="Link3_1">
    <inertial>
      <origin
        xyz="0.078951 -1.1102E-16 1.1102E-16"
        rpy="0 0 0" />
      <mass
        value="0.043996" />
      <inertia
        ixx="1.4695E-05"
        ixy="1.4246E-19"
        ixz="-8.0242E-21"
        iyy="0.00010124"
        iyz="1.2207E-20"
        izz="8.7267E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link3_1_joint"
    type="revolute">
    <origin
      xyz="0.072 0 0"
      rpy="0 0 -1.5708" />
    <parent
      link="Link2_1" />
    <child
      link="Link3_1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="Link_ee">
    <inertial>
      <origin
        xyz="-0.041996 -0.022493 0.00050357"
        rpy="0 0 0" />
      <mass
        value="0.092307" />
      <inertia
        ixx="1.7415E-05"
        ixy="-1.8139E-07"
        ixz="3.0732E-11"
        iyy="9.6059E-05"
        iyz="-5.5841E-12"
        izz="9.4065E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link_ee.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.84314 0.81569 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link_ee.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link_ee_joint"
    type="revolute">
    <origin
      xyz="0.1675 0 0"
      rpy="0 0 -1.5708" />
    <parent
      link="Link3_1" />
    <child
      link="Link_ee" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="Link_E">
    <inertial>
      <origin
        xyz="0.028599 0.00046138 0.00729"
        rpy="0 0 0" />
      <mass
        value="0.023256" />
      <inertia
        ixx="2.0848E-06"
        ixy="7.5146E-09"
        ixz="-6.1806E-09"
        iyy="2.7906E-06"
        iyz="1.1288E-08"
        izz="1.8708E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link_E.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.89412 0.79216 0.73333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link_E.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link_E_joint"
    type="prismatic">
    <origin
      xyz="-0.0074884 -0.037 0"
      rpy="1.5708 0 1.5381" />
    <parent
      link="Link_ee" />
    <child
      link="Link_E" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="Link3_2">
    <inertial>
      <origin
        xyz="0.078951 -1.6653E-16 -1.3878E-17"
        rpy="0 0 0" />
      <mass
        value="0.043996" />
      <inertia
        ixx="1.4695E-05"
        ixy="1.7541E-19"
        ixz="-6.8612E-21"
        iyy="0.00010124"
        iyz="-1.9508E-22"
        izz="8.7267E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link3_2_joint"
    type="revolute">
    <origin
      xyz="0.112 0 0"
      rpy="0 0 -1.5708" />
    <parent
      link="Link2_1" />
    <child
      link="Link3_2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
    <mimic
      joint="Link3_1_joint"
      multiplier="1"
      offset="0" />
  </joint>
  <link
    name="Link_ee_">
    <inertial>
      <origin
        xyz="-0.0036766 -0.02139 0.00050313"
        rpy="0 0 0" />
      <mass
        value="0.085784" />
      <inertia
        ixx="1.7368E-05"
        ixy="-1.8139E-07"
        ixz="4.1281E-11"
        iyy="8.6313E-05"
        iyz="-3.6849E-12"
        izz="8.432E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link_ee_.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.67843 0.76471 0.12549 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link_ee_.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link_ee_joint_"
    type="revolute">
    <origin
      xyz="0.1675 0 0"
      rpy="0 0 -1.5708" />
    <parent
      link="Link3_2" />
    <child
      link="Link_ee_" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="Link2_2">
    <inertial>
      <origin
        xyz="0.061889 8.3267E-17 -0.0001"
        rpy="0 0 0" />
      <mass
        value="0.039322" />
      <inertia
        ixx="4.29E-06"
        ixy="-1.4902E-22"
        ixz="1.0222E-18"
        iyy="4.6463E-05"
        iyz="2.4557E-22"
        izz="4.3641E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link2_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link2_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link2_2_joint"
    type="revolute">
    <origin
      xyz="0 0 0.095554"
      rpy="-1.5708 0 0" />
    <parent
      link="Link1" />
    <child
      link="Link2_2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
    <mimic
      joint="Link2_1_joint"
      multiplier="1"
      offset="0" />
  </joint>
  <link
    name="Link3_1_">
    <inertial>
      <origin
        xyz="0.038951 5.5511E-17 1.1102E-16"
        rpy="0 0 0" />
      <mass
        value="0.043996" />
      <inertia
        ixx="1.4695E-05"
        ixy="2.2449E-20"
        ixz="-7.9908E-21"
        iyy="0.00010124"
        iyz="1.4208E-20"
        izz="8.7267E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_1_.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_1_.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link3_1_joint_"
    type="revolute">
    <origin
      xyz="0.072 0 0"
      rpy="0 0 -1.5708" />
    <parent
      link="Link2_2" />
    <child
      link="Link3_1_" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="Link3_2_">
    <inertial>
      <origin
        xyz="0.038951 -1.1102E-16 -1.3878E-17"
        rpy="0 0 0" />
      <mass
        value="0.043996" />
      <inertia
        ixx="1.4695E-05"
        ixy="5.5336E-20"
        ixz="-7.515E-21"
        iyy="0.00010124"
        iyz="1.8189E-21"
        izz="8.7267E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_2_.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rcm/meshes/Link3_2_.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Link3_2_joint_"
    type="revolute">
    <origin
      xyz="0.112 0 0"
      rpy="0 0 -1.5708" />
    <parent
      link="Link2_2" />
    <child
      link="Link3_2_" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-10"
      upper="10"
      effort="10"
      velocity="10" />
  </joint>
</robot>